registry/donor/forms.py

```diff
@@ -46,3 +47,53 @@ class IgnoreDonorForm(FlaskForm):
 
 class RemoveFromIgnoredForm(FlaskForm):
     rodne_cislo = HiddenField(validators=[DataRequired()])
+
+
+class DonorsOverrideForm(FlaskForm):
+    rodne_cislo = StringField("Rodné číslo", validators=[DataRequired()])
+    first_name = StringField("Jméno")
+    last_name = StringField("Příjmení")
+    address = StringField("Adresa")
+    city = StringField("Město")
+    postal_code = StringField("PSČ", validators=[NumericValidator(5)])
+    kod_pojistovny = StringField("Pojišťovna", validators=[NumericValidator(3)])
+
+    _fields_ = [
+        "rodne_cislo",
+        "first_name",
+        "last_name",
+        "address",
+        "city",
+        "postal_code",
+        "kod_pojistovny",
+    ]
+
+    def validate(self):
+        initial_validation = super(DonorsOverrideForm, self).validate()
+        if not initial_validation:
+            return False
+
+        self._get_field_data()
+        return True
+
+    def init_fields(self, rodne_cislo):
+        override = DonorsOverride.query.get(rodne_cislo)
+
+        if override is not None:
+            for field in self._fields_:
+                data = getattr(override, field)
+                if data is not None:
+                    getattr(self, field).data = data
+
+        self.rodne_cislo.data = rodne_cislo
+
+        return self
+
+    def _get_field_data(self):
+        self.field_data = {}
+        for field in self._fields_:
+            data = getattr(self, field).data
+            if data:
+                self.field_data[field] = data
+            else:
+                self.field_data[field] = None
```

Comment:
Možná by šlo zkrátit na:
```suggestion
            data = getattr(self, field).data or None
```
Tedy pokud je výchozí hodnota pro stringfield prázdný řetězec.

Reply:
Provedeno.